src/fixtures/github/releases/renovate/index.ts

```diff
@@ -0,0 +1,36 @@
+import { renovateReleasesPage1 } from './page1'
+import { renovateReleasesPage10 } from './page10'
+import { renovateReleasesPage11 } from './page11'
+import { renovateReleasesPage12 } from './page12'
```

Comment:
The sorting is a bit weird here. Is this because of Prettier or ESlint?

Reply:
Yeah, it is. This is the usual order in computer related stuff, to be honest. Look at how GitHub sorted the fixture files:

![imagen](https://user-images.githubusercontent.com/2677072/189615813-dc5b4ef8-8985-4c7d-9161-7f587a255e38.png)